docs/tutorial/index.md

```diff
@@ -1,39 +1,39 @@
 ---
-title: Gatsby.js Tutorials
+title: Tutorial Gatsby.js
 disableTableOfContents: true
 ---
 
-Welcome to Gatsby! We’re glad you’re here. The goal of this tutorial is to guide you through setting up and deploying your first Gatsby site using a starter template. As we walk through that process, we’ll introduce some more general web development topics, and go over the underlying structure of a Gatsby site.
+Selamat datang di Gatsby! Kami senang Anda ada di sini. Tujuan dari tutorial ini adalah untuk memandu Anda mengatur dan menyebarkan situs Gatsby Anda menggunakan templat pemula. Saat kita menjalani proses itu, kami akan memperkenalkan beberapa topik pengembangan web yang lebih umum, dan membahas struktur dasar situs Gatsby.
 
-> The full tutorial is intended to be as accessible as possible to people without much web development experience (yet!) — no need to be an expert. If you prefer to jump straight to code, feel free to skip the step-by-step tutorial and see the [quick start](/docs/quick-start/) page.
+> Tutorial lengkap ini dimaksudkan agar dapat diakses oleh orang-orang tanpa banyak pengalaman pengembangan web (belum!) — tidak perlu menjadi seorang ahli. Jika Anda lebih suka melompat langsung ke kode, silakan lewati tutorial langkah demi langkah dan lihat halaman [mulai cepat](/docs/quick-start/).
 
-## Gatsby fundamentals
+## Dasar Gatsby
 
-0.  [Set up your development environment](/tutorial/part-zero/): We'll introduce you to core technologies that power Gatsby, and guide you through setting up your development environment.
-1.  [Get to know Gatsby building blocks](/tutorial/part-one/): Starting new projects, developing, and deploying sites.
-1.  [Introduction to using CSS in Gatsby](/tutorial/part-two/): Explore libraries like Typography.js and CSS Modules.
-1.  [Building nested layouts in Gatsby](/tutorial/part-three/): Layouts are sections of your site that are reused across multiple pages like headers and footers.
+1.  [Menyiapkan lingkungan pengembangan Anda](/tutorial/part-zero/): Kami akan memperkenalkan Anda inti dari teknologi yang memberi kekuatan pada Gatsby, dan memandu Anda menyiapkan lingkungan pengembangan Anda.
+2.  [Mengenal blok bangunan Gatsby](/tutorial/part-one/): Memulai proyek baru, mengembangkan, dan menyebarkan situs.
+3.  [Pengantar menggunakan CSS di Gatsby](/tutorial/part-two/): Menjelajahi pustaka seperti Typography.js dan CSS Modules.
+4.  [Membangun tata letak bersarang di Gatsby](/tutorial/part-three/): Tata letak adalah bagian dari situs Anda yang digunakan kembali di beberapa halaman seperti header dan footer.
```

Comment:
Its better to keep "nested layouts" with italic text. So the text will be : "Membangun _nested layouts_ di Gatsby"